code/modules/surgery/organs/stomach.dm

```diff
@@ -0,0 +1,15 @@
+/obj/item/organ/stomach
+	name = "stomach"
+	icon_state = "stomach"
+	origin_tech = "biotech=4"
+	w_class = WEIGHT_CLASS_NORMAL
+	zone = "chest"
+	slot = "stomach"
+	attack_verb = list("gored", "squished", "slapped")
+	desc = "Onaka ga suite imasu."
+
+/obj/item/organ/stomach/on_life()
+	var/mob/living/carbon/human/H = owner
+
+	if(ishuman(H))
+		H.dna.species.handle_digestion(H)
```

Comment:
It would be neat if this was abstracted more than it is, so you could mix/match unique digestions.

for example, having a fly stomach in a human would allow the human to process nutritious reagents like a fly.